platform/net/routes_searcher_windows.go

```diff
@@ -15,15 +15,26 @@ func NewRoutesSearcher(_ boshlog.Logger, cmdRunner boshsys.CmdRunner, interfaceM
 	return windowsRoutesSearcher{interfaceManager, cmdRunner}
 }
 
-func (s windowsRoutesSearcher) SearchRoutes() ([]Route, error) {
+func (s windowsRoutesSearcher) SearchRoutes(ipv6 bool) ([]Route, error) {
```

Comment:
can we choose `ipv6`, `isIpv6` (or perhaps `isIPv6`)... 

`is_ipv6` seems wrong in that Golang suggest camelCase for variables, as does `IsVersion6` (see `func (ia failingInterfaceAddress) GetIP(IsVersion6 bool) (string, error)`)